tests/_methods/dinov2/test_dinov2.py

```diff
@@ -277,77 +277,31 @@ def check_param_groups() -> None:
 
 class TestDINOv2Args:
     @pytest.mark.parametrize(
-        "scaling, scaling_result",
+        "model_params, expected_method_args",
         [
-            (
-                ScalingInfo(dataset_size=IMAGENET_SIZE, epochs=100),
-                ScalingResult(
-                    output_dim=65536,
-                    start_teacher_temp=0.04,
```

Comment:
Removed `ScalingResult` as method args now only depend on model and not dataset.